library/common.h

```diff
@@ -334,4 +334,23 @@ static inline void mbedtls_xor_no_simd(unsigned char *r,
 #define MBEDTLS_OPTIMIZE_FOR_PERFORMANCE
 #endif
 
+/* Define macros that can be used to disable warnings about unreachable code. */
+#if defined(__clang__)
+
+#define MBEDTLS_PRAGMA(x) _Pragma(#x)
```

Comment:
What's the point of this macro, when there's only a single implementation?

Reply:
Originally I used it in x509_crt, but then realised we could just use `MBEDTLS_PUT_UINT16_BE` instead.